### ISSUE TITLE
Pallet Issues on Samsung Chromebook

### ISSUE THREAD

**Issue by simonh2000** [juliandescottes/piskel#475]
This is mostly a minor issue, but I have been having issues with the pallet on one of my piskel files. The pallet changed, adding multiple variations of what appeared to be the exact same colors, but were actually slightly different. For instance, if I had a gray color it would have different versions of gray on different points on the slider, like pink or blue. 
I just ignored it at first, because it didn't visibly change the hue, but after I deleted the unnecessary colors off my pallet I noticed that some of the pixel colors were leaking - that is, if I had a dark blue pixel next to a white pixel it would make the white pixel slightly blue, and the dark blue pixel a tad lighter.
I used the "Paint all pixels the same color" tool to smooth it back out, but a few hours later I miss-clicked and hit a white pixel while using the "Paint all pixels the same color" tool, _but it didn't paint all white pixels that color._ I have a very small pallet, I started out with one, so this is the same leaking problem as before, but I didn't notice all of those extra pixels had been changed as well because the difference was too slight.

TL;DR My pallet broke and colors are changing the colors around them as if I had used a blur tool on them (Which I didn't, of course).

Do you think this is a browser issue? I'm using Chrome, on a Samsung Chromebook, for reference.

**Comment by juliandescottes** [comment]
So I have already seen reports of this kind of issues with Chromebooks and Piskel. 
From my previous investigations it looks like Context2d.getImageData generates a PNG with random color variations.

The other Chromebook for which this was reported was a XE303C12-A01US. Can you share your Chromebook's specs?

The next step should be to try to isolate the issue outside Piskel and open a ticket on Chromium's bug tracker.  I will try to upload a few test pages to see if we can get a good test case.

**Comment by simonh2000** [comment]
Yes, it's an XE303C12-A01US as well. Thank you for the quick response! ^.^
